### ISSUE TITLE
Compile issue on ios

### ISSUE THREAD

**Issue by morioma** [TruckMovers/cordova-plugin-remote-injection#1]
Hi,
First of all, thank you for this plugin. You know how important this is for achieving external site based solution.
The plugin works as expected on cordova Android 5.1.1. However, I got compile error for cordova ios 4.1.1. 
I included some error message below.

    /Users/digitalsign1/Documents/Stopfinder/Client/cordova/platforms/ios/Stopfinder/Plugins/cordova-plugin-remote-injection/CDVRemoteInjection.m:55:57: error: 
      property 'delegate' not found on object of type 'UIView *'
    notificationDelegate.wrappedDelegate = self.webView.delegate;
                                                        ^
    /Users/digitalsign1/Documents/Stopfinder/Client/cordova/platforms/ios/Stopfinder/Plugins/cordova-plugin-remote-injection/CDVRemoteInjection.m:56:19: error: 
      no visible @interface for 'UIView' declares the selector 'setDelegate:'
    [self.webView setDelegate:notificationDelegate];
     ~~~~~~~~~~~~ ^~~~~~~~~~~
    /Users/digitalsign1/Documents/Stopfinder/Client/cordova/platforms/ios/Stopfinder/Plugins/cordova-plugin-remote-injection/CDVRemoteInjection.m:161:23: error: 
      no visible @interface for 'UIView' declares the selector 'reload'
        [self.webView reload];
         ~~~~~~~~~~~~ ^~~~~~

I tried it on Cordova ios 3.9.2 and it works. Is it currently Cordova ios 4+ not supported?

Any idea on the issue I have? 
Sincerely,

**Comment by bradleyjames** [comment]
Thanks for reporting this.  I'll take a look.

**Comment by bradleyjames** [comment]
Looks like I'm running the 3.9.2 version of the cordova iOS plugin. I'll update and see what I can find.

**Comment by morioma** [comment]
Thanks :) cordova ios 4+ supports the new WKWebView, which should have
better performance. I heard that, when the WKWebview first came out with
iOS 8, it doesn't have ability to load local scripts, so people have
implemented local web server to serve file to overcome the issue. The issue
is fixed on iOS 9. However, I am thinking maybe external website with
injected script won't suffer form the issue at all.

**Comment by bradleyjames** [comment]
I hacked up a solution to this over my lunch break.  It's in the ios40 branch.  There's still some clean up to do but if you like pull it down and see if it fixes your issues, it should.

**Comment by morioma** [comment]
It totally works. Just tried it on simulator and a device with cordova ios 4.1.1. All plugins are working fine. Thanks for the update. Your plugin is concise and essential. Looking forward to the release of the update. 

(Side note: I do notice some big behavior difference between ios 4.1.1 and 3.9.2 regarding clicking, link handling)

**Comment by bradleyjames** [comment]
Released as v0.2.0 to npm.  Thanks for reporting this!
